nnpdf_data/nnpdf_data/commondata/CMS_Z0J_8TEV/metadata.yaml

```diff
@@ -1,75 +1,78 @@
 setname: CMS_Z0J_8TEV
-version: 1
-version_comment: Port of old commondata
+
 nnpdf_metadata:
   nnpdf31_process: DY NC
   experiment: CMS
+
 arXiv:
   url: https://arxiv.org/abs/1504.03511
   journal: Phys. Lett. B749 (2015) 187
 iNSPIRE:
-  url: ''
+  url: https://inspirehep.net/literature/1359450
 hepdata:
-  url: None
-  version: -1
+  url: https://www.hepdata.net/record/ins1359450
+  version: 1
+
+version: 2
+version_comment: Implementation in the new format
+
 implemented_observables:
 - observable_name: PT-Y
   observable:
     description: Drell-Yan Transverse Momentum Distribution
     label: CMS $Z$ $p_T$ 8 TeV $(p_T^{ll},y_{ll})$
-    units: ''
-  process_type: EWK_PTRAP
-  tables: []
-  npoints: []
+    units: '[fb]'
+  process_type: DY_NC_PTRAP
+  tables: [2, 4]
   ndata: 50
   plotting:
-    kinematics_override: ewk_ptrap_sqrt_scale
+    kinematics_override: identity
     dataset_label: CMS $Z$ $p_T$ 8 TeV $(p_T^{ll},y_{ll})$
     y_label: $d\sigma_{Z/\gamma^{*}}/d\eta$ (fb)
     figure_by:
-    - k2
-    plot_x: k1
-  kinematic_coverage:
-  - k1
-  - k2
-  - k3
+    - abs_y
+    plot_x: pT
+  kinematic_coverage: [pT, abs_y, m_Z2]
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
+      abs_y:
+        description: Absolute rapidity of the Z boson
+        label: $|y_Z|$
         units: ''
+      pT:
+        description: Transverse momentum of the Z boson
+        label: $p_T^Z$
+        units: GeV
+      m_Z2:
+        description: Mass of the Z boson squared
+        label: m_Z2
+        units: GeV$^2$
     file: kinematics_PT-Y.yaml
-  theory:
-    conversion_factor: 1.0
-    operation: 'null'
-    FK_tables:
-    - - CMSZDIFF12-CMSZDIFF12-BIN1_ptZ
-      - CMSZDIFF12-CMSZDIFF12-BIN2_ptZ
-      - CMSZDIFF12-CMSZDIFF12-BIN3_ptZ
-      - CMSZDIFF12-CMSZDIFF12-BIN4_ptZ
-      - CMSZDIFF12-CMSZDIFF12-BIN5_ptZ
-  data_uncertainties: []
+  data_central: data_PT-Y.yaml
+  data_uncertainties: [uncertainties_PT-Y.yaml]
   variants:
+    sys_10:
+      data_uncertainties:
+      - uncertainties_PT-Y_sys_10.yaml
     legacy:
       data_uncertainties:
       - uncertainties_legacy_PT-Y.yaml
     legacy_NNPDF31:
       data_uncertainties:
-      - uncertainties_PT-Y_sys_NNPDF31.yaml
+      - uncertainties_legacy_PT-Y_sys_NNPDF31.yaml
     legacy_10_NNPDF31:
       data_uncertainties:
-      - uncertainties_PT-Y_sys_10_NNPDF31.yaml
+      - uncertainties_legacy_PT-Y_sys_10_NNPDF31.yaml
     legacy_10:
       data_uncertainties:
```

Comment:
Do we need to keep all these? For backward compatibility I guess we will at least want to keep `legacy_10` since that's what has been used in fits so far, but I don't think any of the other have?

Reply:
I agree that only `legacy_10` was used for the fits However, since the new implementation does not match with the old one, I would keep both just in case. Do you think this is a problem?